src/border_agent/border_agent.cpp

```diff
@@ -320,6 +325,11 @@ void BorderAgent::UnpublishEpskcService()
     });
 }
 
+void BorderAgent::AddEphemeralKeyChangedCallback(EphemeralKeyChangedCallback aCallback)
+{
+    mEphemeralKeyChangedCallbacks.emplace_back(std::move(aCallback));
```

Comment:
```suggestion
    mEphemeralKeyChangedCallbacks.push_back(std::move(aCallback));
```
nit: no need to use `emplace_back`.

Reply:
Done